tsdb/shard.go

```diff
@@ -447,6 +447,17 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]*FieldCreate,
 
 	// get the shard mutex for locally defined fields
 	for _, p := range points {
+		// verify the tags and fields
+		tags := p.Tags()
```

Comment:
This isn't the right place to reject this as it will fail the whole batch instead of the point.